commercial/app/commercial/CommercialLifecycle.scala

```diff
@@ -93,7 +118,7 @@ trait CommercialLifecycle extends GlobalSettings with Logging with ExecutionCont
       val feedName = fetcher.feedMetaData.name
       val jobName = mkJobName(feedName, "fetch")
       Jobs.deschedule(jobName)
-      Jobs.scheduleEveryNMinutes(jobName, 15) {
+      Jobs.scheduleEveryNMinutes(jobName, 1) {
```

Comment:
Is there a reason that we suddenly want to start fetching the feeds every minute? Seems slightly excessive; if a fetch takes longer than 1 minute then we will end up accumulating scheduled jobs.